dandi/tests/fixtures.py

```diff
@@ -390,6 +395,87 @@ def zarr_dandiset(new_dandiset: SampleDandiset) -> SampleDandiset:
     return new_dandiset
 
 
+@pytest.fixture()
+def create_video_files(tmp_path):
+    video_paths = []
+    import cv2
+
+    video_path = tmp_path / "video_files"
+    video_path.mkdir()
+    for no, ext in enumerate([".avi", ".avi"]):
```

Comment:
Are these file extensions supposed to be the same?  If yes, just do `ext = ".avi"` and `for no in range(2):`.

Reply:
yes, correct. Initially I had it setup to work with multiple extensions but different ones require codecs so I dropped that. fixed here https://github.com/dandi/dandi-cli/pull/841/commits/a43c81a7aa990ddb66def6b9f93633c410b6910c